src/main/groovy/wooga/gradle/release/utils/ReleaseNotesGenerator.groovy

```diff
@@ -171,4 +169,84 @@ class ReleaseNotesGenerator {
             return false
         }
     }
+
+    private List<PacketDependency> fetchTemplateDependencies(GHRepository hub, String packageId, ReleaseVersion version) {
+
+        String[] paths = ["$packageId/paket.template", "paket.template"]
+        String content
+
+        paths.each {
+            if (!content) {
+                content = GetTemplateContent(hub, it, version)
+            }
+        }
+
+        if (!content) {
+            return []
+        }
+
+        PaketTemplateReader templateReader = new PaketTemplateReader(content)
+        return templateReader.getDependencies()
+
+    }
+
+    private String GetTemplateContent(GHRepository hub, String path, ReleaseVersion version) {
+        try {
+            def content = hub.getFileContent(path, "v$version.version").read().text
+            return content
+        }
+        catch (IOException exception) {
+            logger.error("could not find paket template file at $path $exception.message")
```

Comment:
I would log this as an `info` since it's not an issue for projects without `paket.template` files. The exception method should be logged as `debug`